### ISSUE TITLE
A tiny question on a coefficient in training code

### ISSUE THREAD

**Issue by xuanlongORZ** [gengshan-y/monodepth-uncertainty#1]
Hi Gengshan, I really like this work.
However, when I checked the loss part in your training code (in the drive), I have a small question on 'alpha = 15'. Is it related to the sigma design in the paper, can you explain it a bit in detail? Because the sigma part in the paper (lines below eq.4) is also confused me. Thank you :>.

**Comment by gengshan-y** [comment]
You are right. It is a misuse of notations in the code and `alpha =1/(2*sigma^2)`.
Fig.6 might help understanding sigma. Smaller sigma (larger alpha) corresponds to gaussian distributions with lower variance in Fig. 6. Sigma approaching zero corresponds to the standard one-hot labels.
